assets/scripts/admin/statistics.js

```diff
@@ -264,24 +264,10 @@ LS.Statistics2 = function () {
         $('#statisticsoutput .row').first().find('.chartjs-container').loadGraph();
     }
 
-    $('#generalfilters-chevron').click(function () {
-        toggleSection($('#generalfilters-chevron'), $('#statisticsgeneralfilters'));
-    });
-
-    $('#responsefilters-chevron').click(function () {
-        toggleSection($('#responsefilters-chevron'), $('#filterchoices'));
-    });
-
-    $('#statistics-render-chevron').click(function () {
-        toggleSection($('#statistics-render-chevron'), $('#statisticsoutput'));
-    });
-
     $('#generate-statistics').submit(function () {
-        hideSection($('#generalfilters-chevron'), $('#statisticsgeneralfilters'));
-        hideSection($('#responsefilters-chevron'), $('#filterchoices'))
+        hideSection('general-filters-item-body');
```

Comment:
Did we really need to collapse or do any HTML update for PDF and XLSX export ? 

Only needed for HTML, no ?